CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### 1.18.0
+- Feat: Do not suggest propertyNames if doNotSuggest is true [#1045](https://github.com/redhat-developer/yaml-language-server/pull/1045)
```

Comment:
I think that #1028 (fix enum values to be unique) needs to be included, as well as #985 (multipleof does not work for floats)